frame/election-provider-support/solution-type/fuzzer/src/compact.rs

```diff
@@ -8,6 +8,7 @@ fn main() {
 		VoterIndex = u32,
 		TargetIndex = u32,
 		Accuracy = Percent,
+		SizeBound = frame_support::traits::ConstU32::<100_000>, //TODO: unsure about size
```

Comment:
```suggestion
		MaxVoters = frame_support::traits::ConstU32::<100_000>,
```

it is okay for this fuzzer.